### ISSUE TITLE
Logshark fails to persist any data from Backgrounder logs

### ISSUE THREAD

**Issue by FLuGonz** [tableau/Logshark#44]
Hello,

since April we are having problems with the backgrounder workbook. We run Logshark everyday and upload the the results to the server. However, the backgrounder workbooks is not updating since April. The following error message is repeated regularly in the backgrounder logs:

> 2018-05-31 02:13:31,516 fr02513vma_18053100100395_tabservprodlogzip (null) [1] INFO Logshark.Plugins.Backgrounder.Backgrounder - Failed to persist any data from Backgrounder logs!

The rest of the workboos we check on a weekly basis are working fine and this issue only affects the backgrounder. We are currently using Tableau 10.5.

Please let me know if you need any additional information.

Thanks!

**Comment by jmangue** [comment]
Hi @FLuGonz -- Thanks for your report.  I believe this issue is due to some log format changes that happened when a new field was added to the backgrounder logs.  This issue will be fixed in the next release.  I will leave this open as a bug for now and update it when the new release is out.

**Comment by FLuGonz** [comment]
Hi @jmangue Thanks for the heads-up! We'll wait for the next release then.

**Comment by jmangue** [comment]
Hi @FLuGonz, we just released a new version that should fix this... please upgrade to https://github.com/tableau/Logshark/releases/tag/v2.1 and let us know if you are still having problems!

**Comment by FLuGonz** [comment]
Hello @jmangue 
thanks for your answer. Actually we have uploaded Logshark to the new version but we are still having issues. Whenever we try to run logshark, the command prompt shows the following message:

loading logstark user configuration...

After that nothing happens. The logs keep on repeating the same message, which is as follows:

2018-06-28 13:51:44,522 (null) (null) [1] INFO Logshark.RequestModel.Config.LogsharkConfigReader - Loading Logshark user configuration..
2018-06-28 13:51:44,553 (null) (null) [1] DEBUG Logshark.RequestModel.Config.LogsharkConfigReader - Loaded user configuration!

The Logshark.config was updated accordingly after the update and the information should be correct. Any ideas what could be the problem here? Thanks!
